tab.fish

```diff
@@ -1,7 +1,6 @@
-# Open the current directory in a new tab
```

Comment:
There's a much more robust `tab` implementation over at [plugin-tab](https://github.com/oh-my-fish/plugin-tab). Should we remove this one and point them there instead?